pwa-devdocs/src/_drafts/page-builder/creating-pb-components/index.md

```diff
@@ -23,16 +26,19 @@ As we require some additional steps compared to a normal we have some additional
 `__tests__` & `index.js` are just part of PWA Studio, and won't be discussed here.
 
 ### `configAggreagtor.js`
```

Comment:
I hope this isn't the spelling for the actual file

Reply:
This was a WIP produced by myself and thus was written poorly :) 

The file is `configAggregator.js`